elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -313,11 +321,29 @@ public FilterExpression visit(ComparisonNode node, Class entityType) {
                 } else {
                     return new NotFilterExpression(new Predicate(path, Operator.IN, values));
                 }
+            } else if (op == isnullOp) {
```

Comment:
@aklish could you explain a bit on why it's too late here?  I think these if..else statements are just matching the `op` and then assign correct handler for it.